Libraries/LibWeb/HTML/TraversableNavigable.cpp

```diff
@@ -1423,6 +1423,20 @@ void TraversableNavigable::set_emulated_position_data(Geolocation::EmulatedPosit
     m_emulated_position_data = data;
 }
 
+// https://www.w3.org/TR/webdriver-bidi/#webdriver-bidi-emulated-max-touch-points
+Optional<WebIDL::Long> TraversableNavigable::emulated_max_touch_points() const
+{
+    VERIFY(is_top_level_traversable());
```

Comment:
The emulated max touch points also has very specific spec text that is not implemented here. It won't be a getter/setter on the traversable, it will have to reach into webdriver session objects.

IMO it'd be better to just leave the emulated part as a FIXME. We haven't started a webdriver bidi implementation at all yet, so we don't know what this is going to look like.